romana-install/group_vars/vagrant

```diff
@@ -1,7 +1,27 @@
 ram_size:
   controller:
     devstack: 4096
-    kubernetes: 1024
+    kubernetes: 2048
   compute:
     devstack: 1024
```

Comment:
Should we just increase devstack compute to 2048 as well? Seem like it may usually require more (rather than less) memory compared to k8s compute.